### PR TITLE
Grant new save (fixes breakage)

### PR DESCRIPTION
**Pull request by GrantHartenstine**

### CODE REVIEW

File: src/main/java/utility/Main.java [hunk 1]
```diff
@@ -12,12 +12,12 @@
 	 */
 	public static void main(String[] args) {
 		//If we have a command line arg run the GUI environment
-		if (args.length > 0) {
+		//if (args.length > 0) {
```
Comment (MattFossett) [inline src/main/java/utility/Main.java]: Remove these comments in main

**Review by MattFossett — CHANGES_REQUESTED** [review]
The comments in main make it so CLI won't run, please remove

**Comment by lndeaver** [comment]
I helped grant fix some merging conflicts that we had to revert back to for this after things were added to develop. His imports were too specific and didn't include things that were needed for other functions/uses.

**Review by MattFossett — APPROVED** [review]
Changes made, testing shows that it is just the buttons added. Approved.

**Pull request merged**